README.md

```diff
@@ -96,7 +97,7 @@ $ ./minikube_demo_setup.sh
 $ ./minikube_demo_setup.sh -sp
 
 # If you want to restart only autotune with the specified docker image
-$ ./minikube_demo_setup.sh -r -i [autotune docker image]
+$ ./minikube_demo_setup.sh -r -i [autotune operator image] -o [autotune optuna image]
```

Comment:
Should we mention about default images that will be used, as these options are optional?

Reply:
Mentioned the default images in the first command above.